systemtest/src/test/java/io/strimzi/systemtest/kafka/KafkaNodePoolST.java

```diff
@@ -229,7 +229,7 @@ void testKafkaNodePoolBrokerIdsManagementUsingAnnotations(ExtensionContext exten
 
         Kafka kafka = KafkaTemplates.kafkaPersistent(testStorage.getClusterName(), 1, 1)
             .editOrNewMetadata()
-                .addToAnnotations(Annotations.ANNO_STRIMZI_IO_NODE_POOLS, "enabled")
+                .addToAnnotations(Map.of(Annotations.ANNO_STRIMZI_IO_NODE_POOLS, "enabled", Annotations.ANNO_STRIMZI_IO_SKIP_BROKER_SCALEDOWN_CHECK, "true"))
```

Comment:
Maybe adding comment here would be helpful for the one who will fix those two tests, thanks! :)

Reply:
I was wondering if I should mention about this failing test in this issue https://github.com/strimzi/strimzi-kafka-operator/issues/9134 itself? I have left a comment about this issue already in the other failing test.

Reply:
Wdyt?

Reply:
I think it is a good idea, We can change these test at the same time anyway.

Reply:
Okay, I will update the issue then, Thanks @henryZrncik

Reply:
Done